### DISCUSSION TITLE
Help install for termux

### DISCUSSION THREAD

**Discussion by ExtinctDrift19** [sdkman/sdkman-cli#1047] [Ideas]
Site https://sdkman.io/ add instruction sdkman install for termux

**Comment by ExtinctDrift19** [comment]
link termux https://github.com/termux/termux-packages About
A build system and primary set of packages for Termux.

**Comment by ExtinctDrift19** [comment]
link termux github.com/termux

**Comment by helpermethod** [comment]
Hi @ExtinctDrift19!

While it's possible to install SDKMAN! on termux (I've done this myself), it is only of limited use, as e.g. the Java distributions you can install are broken after installation, possibly because the architecture is not supported.

**Comment by ExtinctDrift19** [comment]
fix termux pkg install openjdk-17

**Comment by tripleo1** [comment]
It might work if you do a chroot install. I don't know if they still have that feature.

I remember a long time ago I ran a arm native jdk on termux.
